contrib/sssd.spec.in

```diff
@@ -1079,7 +1079,7 @@ install -D -p -m 0644 %{SOURCE1} %{buildroot}%{_sysusersdir}/sssd.conf
 
 %if %{build_passkey}
 %files passkey
-%attr(755,%{sssd_user},%{sssd_user}) %{_libexecdir}/%{servicename}/passkey_child
+%attr(755,root,root) %{_libexecdir}/%{servicename}/passkey_child
```

Comment:
I think `%attr(755,root,root)` is a default and can be just skipped.
See `p11_child` for example.

Reply:
Yes, and thanks!